packages/ws-server/src/webSocketServer.ts

```diff
@@ -213,6 +215,10 @@ app.ws.use(async (ctx) => {
 
         ctx.websocket.send(JSON.stringify(response));
     });
+
+    setInterval(async () => {
```

Comment:
Q: Does this extend the connection time of some subscribers. Will this keep their session alive vs closing after a while of no message activity?

Reply:
The current TTL period is not extended when a message is sent. Maybe we should change it to be based on the last subscription message?

Reply:
Yeah that sounds appreciate.
If ttl is 15 mins and the last message shows up at minute 14, ttl should get reset to 15 mins again.